docs/modules/ROOT/pages/tutorial/06_derived-roles.adoc

```diff
@@ -0,0 +1,29 @@
+include::partial$attributes.adoc[]
+
+= Derived roles
+
+NOTE: The policies for this section can be found https://github.com/Sambigeara/cerbos/tree/docs/main/docs/modules/ROOT/examples/tutorial/07-derived-roles/cerbos[on Github].
```

Comment:
```suggestion
NOTE: The policies for this section can be found link:{app-github-url}/tree/docs/main/docs/modules/ROOT/examples/tutorial/07-derived-roles/cerbos[on GitHub].
```

The attributes are defined here: https://github.com/cerbos/cerbos/blob/main/docs/modules/ROOT/partials/attributes.adoc

Reply:
Oh brill, thank you! And good spot on the fork (was using for testing), I thought I got them all 😅 . I'll check and adjust to the attribute.

Sorry for the back and forth here, like a game of antora tennis! Hopefully this'll get it over the line